client/components/SocialClub/SocialClubPage.js

```diff
@@ -67,18 +61,19 @@ class SocialClubPage extends Component {
     });
   }
 
-  join(details={'club_id': 3, 'email': 'ig@uk.com'}){
+  join(){
+    let details = {'club_id': this.props.club.id, 'email': this.props.user.app_user.email};
     this.props.joinClub(details)
-    .then(() => {
-      toastr.success('You have successfully joined this Club. You will be notified of new events');
-    })
-    .catch((error) => {
-      toastr.error('Aww! Something went wong');
-    });
+      .then(() => {
+        toastr.success('You have successfully joined this Club. You will be notified of new events');
+      })
+      .catch((error) => {
+        toastr.error('Aww! Something went wong');
+      });
   }
 
   render(){
-    const { name, id, featured_image, events, created_by, created_on, description, venue, date, time } = this.props.club;
+    const { name, featured_image, events, description} = this.props.club;
```

Comment:
A space is needed before the closing curly brace.

Reply:
Ok.. noted